README.md

```diff
@@ -14,7 +14,7 @@ often!
 
 ## Who are we and what are we working on?
 
-There are currently 37 of us in the Android guild. As an Android Engineer,
+There are currently 32 of us in the Android guild. As an Android Engineer,
```

Comment:
if somebody could sense check that'd be grand :D

Reply:
seems legit! :D